tests/test_prometheus_connect.py

```diff
@@ -23,6 +23,11 @@ def test_metrics_list(self):
         """Check if setup was done correctly."""
         metrics_list = self.pc.all_metrics()
         self.assertTrue(len(metrics_list) > 0, "no metrics received from prometheus")
+        # Checking that the results of all_metrics() and get_label_values("__name__") are the same.
+        self.assertEqual(metrics_list, self.pc.get_label_values("__name__"))
+        # Check for the "job" label.
+        label_values = self.pc.get_label_values("job")
+        self.assertTrue(len(label_values) > 0, "no metrics received from prometheus")
```

Comment:
Here I added a check for this label(job) for your test server (http://demo.robustperception.io:9090/). I am comparing the number of values to zero. If you need to compare the result with the current value and it will not change over time, I can make an edit.
Current value for "job" label: ['alertmanager', 'node', 'prometheus', 'pushgateway']